### DISCUSSION TITLE
Pg_restore unsupported version (1.14) in file header

### DISCUSSION THREAD

**Discussion by Rccfrz** [dbeaver/dbeaver#9172] [Q&A] [answered]
Hello, 

I am trying to restore a database, but I get an error that Pg_restore unsupported version (1.14) in file header 

host is local host, port is 5432, Dbeaver is version 7.1.1
version pg_restore (PostgreSQL) 11.7 (Debian 11.7-0+deb10u1)
I work with a Chromebook 

How can I fix that?

(I should probably add that I am a superbeginner  - installed postgres and Dbeaver 2 days ago)

Let me know!

**Comment by kseniiaguzeeva** [comment, marked as answer]
As I see it is not DBeaver issue, please try to use a client that matches the database version.

**Comment by POG2I** [comment]
It is 100% a DBeaver issue   

The binary it's trying to use is: AppData\Roaming\DBeaverData\drivers\clients\postgresql\win\15\pg_restore.exe 

So somehow this is not updated....
